.github/workflows/cmake.yml

```diff
@@ -20,9 +20,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+        cache: 'pip'
 
     - name: Install dependencies
-      run: sudo apt-get install libdw-dev libunwind-dev gfortran
+      run: |
+        sudo apt-get install libdw-dev libunwind-dev gfortran python-pybind11
```

Comment:
Looks like it couldn't find the `python-pybind11` package. Not sure if it's redundant with the `pip install` command below.

Reply:
That was left over from an earlier attempt. The `pip install` command is what should be used. I've removed the apt install for pybind11.